src/main/java/com/mparticle/kits/MParticleAutopilot.java

```diff
@@ -34,10 +33,9 @@ public AirshipConfigOptions createAirshipConfigOptions(Context context) {
         SharedPreferences preferences = context.getSharedPreferences(PREFERENCE_NAME, Context.MODE_PRIVATE);
 
         AirshipConfigOptions.Builder optionsBuilder = new AirshipConfigOptions.Builder()
-                .setGcmSender(preferences.getString(GCM_SENDER, null))
```

Comment:
No longer need the sender ID since we are not actually requesting a token from GCM/FCM anymore.